index.html

```diff
@@ -76,50 +80,55 @@ <h2>Multi-Post
 				<li><a href="">javascript</a></li>
 			</ul>
 
-			<button class="btn">See Project</button>
+			<button type="button" class="btn">See Project</button>
+		</div>
 		</div>
 
-		<div>
-			<img src="/img/tonic2.svg" alt="">
-			<h2>TONIC</h2>
-			<ul class="work-exp">
-				<li>Canopy</li>
-				<li><img src="/img/circle.svg" alt=""></li>
-				<li>Back End Dev</li>
-				<li><img src="/img/circle.svg" alt=""></li>
-				<li>2015</li>
-			</ul>
-			<p>A daily selection of privately personalized reads; 
-				no accounts or sign-ups required.</p>
-      <ul class="lang">
-				<li><a href="">html</a></li>
-				<li><a href="">css</a></li>
-				<li><a href="">javascript</a></li>
-			</ul>
+		<div id="work_3">
+			<div class="img"><img src="/img/tonic2.svg" alt=""></div>
+		    <div class="content_work">
+				<h2>TONIC</h2>
+				<ul class="work-exp">
+					<li>Canopy</li>
+					<li><img src="/img/circle.svg" alt=""></li>
+					<li>Back End Dev</li>
+					<li><img src="/img/circle.svg" alt=""></li>
+					<li>2015</li>
+				</ul>
+				<p>A daily selection of privately personalized reads; 
+					no accounts or sign-ups required.</p>
```

Comment:
- [ ] [**OPTIONAL**]These link , in this section should not be only one , kindly make sure you change that

 
![Screenshot from 2023-01-21 01-59-35](https://user-images.githubusercontent.com/80458199/213827525-73c8668a-2810-4529-9e12-d9500eeb39fd.png)